### PR TITLE
Update Firefox to 61 and geckodriver to 0.22.0

### PR DESCRIPTION
**Pull request by stephendonner**
I still need to test this locally; is there an internal redash instance I can point the tests too?

### CODE REVIEW

**Comment by jrbenny35** [comment]
@stephendonner not quite, you just have to clone getredash/redash, run docker-compose, and then run ```make setup-redash``` and ```make docker-ui-tests```.

**Comment by stephendonner** [comment]
Sorry, I took a look the other day, but need more focused time (and probably sync up with one of you) to make sure I'm testing this correctly w/redash.

**Comment by jrbenny35** [comment]
We should probably update the Firefox version to match. I think 61 would work.

**Comment by stephendonner** [comment]
Super sorry for the delay; I finally got this all set up using redash/Docker, etc., and now I think I'm hitting/needing https://github.com/mozilla/redash-ui-tests/pull/80.

Full efforts and failures here: https://gist.github.com/stephendonner/c305ddd315f7ec66cb0c5a83fa03b9b6

**Comment by jrbenny35** [comment]
@stephendonner *sigh* looks like there is a new redash release. We will have to revisit this once we update our tests.

**Comment by jrbenny35** [comment]
I don't know why circleci is detecting this as a version 1.0 config when there are no config file changes..

File: Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ RUN dependencies=' \
         ca-certificates \
         curl \
         firefox \
-		python3.6 \
+		    python3.6 \
```
Comment (jrbenny35) [inline Dockerfile]: Can you just remove this change and we can merge. It should be fine on master.

**Comment by jrbenny35** [comment]
Actually, can we get an update to geckodriver 0.23??

**Comment by stephendonner** [comment]
@jrbenny35 done!

**Pull request merged**